app/javascript/spec/domain-form/domain-form.spec.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import fetchMock from 'fetch-mock';
+import { act } from 'react-dom/test-utils';
+import { componentTypes as components } from '@data-driven-forms/react-form-renderer';
+import schema from '../../components/domain-form/domain-form.schema';
+import NamespaceForm from '../../components/domain-form';
+import { mount } from '../helpers/mountForm';
+import '../helpers/miqFlashLater';
+import '../helpers/miqSparkle';
+import '../helpers/addFlash';
+import '../helpers/sprintf';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+describe('Domain form component', () => {
+  const flashSpy = jest.spyOn(window, 'add_flash');
+  let wrapper;
+
+  const defaultProps = {
+    id: 2222,
+  };
+
+  beforeEach(() => {
+    wrapper = undefined;
```

Comment:
Wrapper is already undefined. You should reset its value in `afterEach` instead.